api-compact.yaml

```diff
@@ -300,83 +292,6 @@ x-amazon-apigateway-integration-with-enterprise-customer-catalog-uuid-and-course
-
-    # /v1/enterprise_catalogs/{uuid}/
-    enterpriseCustomerCatalogByUuid:
-        get:
-          produces: *produces
-          parameters:
-            - *auth_header
-            - *uuid_parameter
-            - *page_qs_parameter
-            - *page_size_qs_parameter
-          operationId: "get_enterprise_customer_catalog_by_uuid"
-          responses: *responses
-          x-amazon-apigateway-integration:
-            <<: *apigateway_integration_enterprise_customer_catalog_detail
-            uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/enterprise_catalogs/{uuid}/"
-
-    # /v1/enterprise_catalogs/{uuid}/course_runs/{course_run_id}/
-    enterpriseCustomerCatalogCourseRunByUuid:
-        get:
-          produces: *produces
-          parameters:
-            - *auth_header
-            - *uuid_parameter
-            - *course_run_id_parameter
-            - *page_qs_parameter
-            - *page_size_qs_parameter
-          operationId: "get_enterprise_customer_catalog_course_run_by_id"
-          responses: *responses
-          x-amazon-apigateway-integration:
-            <<: *apigateway_integration_with_enterprise_customer_catalog_uuid_and_course_run_id_parameters
-            uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/enterprise_catalogs/{uuid}/course_runs/{course_run_id}/"
-
-    # /v1/enterprise_catalogs/{uuid}/courses/{course_key}/
-    enterpriseCustomerCatalogCourseByKey:
-        get:
-          produces: *produces
-          parameters:
-            - *auth_header
-            - *uuid_parameter
-            - *course_key_parameter
-            - *page_qs_parameter
-            - *page_size_qs_parameter
-          operationId: "get_enterprise_customer_catalog_course_by_key"
-          responses: *responses
-          x-amazon-apigateway-integration:
-            <<: *apigateway_integration_with_enterprise_customer_catalog_uuid_and_course_key_parameters
-            uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/enterprise_catalogs/{uuid}/courses/{course_key}/"
-
-    # /v1/enterprise_catalogs/{uuid}/programs/{program_uuid}/
-    enterpriseCustomerCatalogProgramByUuid:
-        get:
-          produces: *produces
-          parameters:
-            - *auth_header
-            - *uuid_parameter
-            - *program_uuid_parameter
-            - *page_qs_parameter
-            - *page_size_qs_parameter
-          operationId: "get_enterprise_customer_catalog_program_by_uuid"
-          responses: *responses
-          x-amazon-apigateway-integration:
-            <<: *apigateway_integration_with_enterprise_customer_catalog_uuid_and_program_uuid_parameters
-            uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/enterprise_catalogs/{uuid}/programs/{program_uuid}/"
-
     # /v1/enterprise-customer/{uuid}/course_enrollments/
     enterpriseCustomerCourseEnrollments:
```

Comment:
We still need to have this V1 API endpoint in the enterprise, could you please recall why?

Reply:
`enterprise-customer` - this is another API endpoint, its should be left
I was remove only `enterprise_catalogs`

Reply:
Got it, thanks